README.md

```diff
@@ -1 +1,79 @@
-# libuast
\ No newline at end of file
+# libuast
+
+## Installation
+
+### 1. Build/Install C++ API
+
+```
+cmake .
+make
+make install
+```
+
+### 2.Install python bindings
+
+```
+## move to bindings
+cd bindings
+cmake .
+make
+make install-python
+```
+
+## Usage
+
+### C++
+
+```c++
+#include <iostream>
+#include <libuast/node.h>
+
+void example()
+{
+  auto root = Node("compilation_unit");
+  root.add_role(2);
+
+  auto node1 = Node("class");
+  auto node2 = Node("identifier");
+  node2.add_role(1);
+
```

Comment:
I will update the README to reference external files, right now this example is in too many places!